src/Microsoft.ML.TimeSeries/SequentialTransformerBase.cs

```diff
@@ -190,7 +227,7 @@ public void ProcessWithoutBuffer(ref TInput input, ref TOutput output)
         internal readonly string OutputColumnName;
         private protected ColumnType OutputColumnType;
 
-        public bool IsRowToRowMapper => false;
+        public bool IsRowToRowMapper => true;
```

Comment:
In reality TS is a rowtorowmapper but we will just pretend it is not so that it does not work in the other prediction engine. In TS we will recognize this is actually rowtorowmapper covered by a custom interface that extends IRowToRowMapper. Then we go over transformers one by one in the chain and call GetRow on all of them to get the final transformer. How is this supposed to invalidate check-pointing or cloning? 

---
In reply to: [236009857](https://github.com/dotnet/machinelearning/pull/1618#discussion_r236009857) [](ancestors = 236009857,236003674,235555841,235555650)

Reply:
Done.